api/groups/baseBlockGroup.go

```diff
@@ -118,3 +118,83 @@ func (group *blockGroup) byNonceHandler(c *gin.Context) {
 
 	c.JSON(http.StatusOK, blockByNonceResponse)
 }
+
+func (group *blockGroup) alteredAccountsByNonceHandler(c *gin.Context) {
+	shardID, err := shared.FetchShardIDFromRequest(c)
+	if err != nil {
+		shared.RespondWith(
+			c,
+			http.StatusBadRequest,
+			nil,
+			apiErrors.ErrCannotParseShardID.Error(),
+			data.ReturnCodeRequestError,
+		)
+		return
+	}
+
+	nonce, err := shared.FetchNonceFromRequest(c)
+	if err != nil {
+		shared.RespondWith(
+			c,
+			http.StatusBadRequest,
+			nil,
+			apiErrors.ErrCannotParseNonce.Error(),
+			data.ReturnCodeRequestError,
+		)
+		return
+	}
+
+	options, err := parseAlteredAccountOptions(c)
+	if err != nil {
+		shared.RespondWithValidationError(c, apiErrors.ErrBadUrlParams, err)
+		return
+	}
+
+	blockByNonceResponse, err := group.facade.GetAlteredAccountsByNonce(shardID, nonce, options)
+	if err != nil {
+		shared.RespondWith(c, http.StatusInternalServerError, nil, err.Error(), data.ReturnCodeInternalError)
```

Comment:
`shared.RespondWithInternalError`

Reply:
I have no outer and inner error to pass as asrguments for that func, so I would leave it as it is